src/App.tsx

```diff
@@ -1,24 +1,136 @@
-/* eslint-disable max-len */
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
-import { UserWarning } from './UserWarning';
+import React, { useEffect, useState } from 'react';
+import { createTodo, deleteTodo, getTodos } from './api/todos';
+import { Header } from './Components/Header';
+import { Main } from './Components/Main';
+import { Footer } from './Components/Footer';
+import { Todo } from './types/Todo';
+import { Filter } from './types/Filter';
+import { Errors } from './types/Errors';
 
-const USER_ID = 0;
+const USER_ID = 10307;
 
 export const App: React.FC = () => {
-  if (!USER_ID) {
-    return <UserWarning />;
-  }
+  // if (!USER_ID) {
+  //   return <UserWarning />;
+  // }
+
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [errors, setErrors] = useState<Errors | null>(null);
+  const [filter, setFilter] = useState<Filter>(Filter.All);
```

Comment:
TS already knows about that, bc you put an initial value
```suggestion
  const [filter, setFilter] = useState(Filter.All);
```